pkg/security/secl/rules/fim_unix.go

```diff
@@ -0,0 +1,55 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+//go:build unix
+
+// Package rules holds rules related files
+package rules
+
+import (
+	"fmt"
+	"strings"
+)
+
+type expandedRule struct {
+	id   string
+	expr string
+}
+
+func expandFim(baseID, baseExpr string) []expandedRule {
+	if !strings.Contains(baseExpr, "fim.write.file.") {
```

Comment:
Should we also handle something like `fim.read.file` (or just  `fim.file`?) to generate rules without the open flags?

Reply:
We are going to start with write only rules, because the output open rule has the flags check that is useful for approvers. If the need arises for read rules in the future we can come back to this